pkg/test/path_test.go

```diff
@@ -0,0 +1,18 @@
+package test_test
```

Comment:
A combination of being new to Go and mimicking what's already here.

Starting with the fact that this file is "tests for the package tests", I noticed there are other places in zq.git that declare "tests for package" as "pkg_test", like the following:

- https://github.com/brimsec/zq/blob/19fe752550d6354dab57909b4641341a2a996c2a/proc/sort_test.go#L1
- https://github.com/brimsec/zq/blob/19fe752550d6354dab57909b4641341a2a996c2a/pkg/nano/span_test.go#L1

And, if someone wanted to import "pkg/test", they don't strictly need access to that package's unit tests.

These were my reasonings.

Reply:
The proc pacakge usage is to avoid a circular dependency that we should remove in the future; I don't see an obvious reason for the nano package to do so. So, in this PR, let's stick with the go convention and use `package test`.

Reply:
Done at 1d8941b .